src/Components/AdvancedSearch/AdvancedSearch.styles.ts

```diff
@@ -0,0 +1,65 @@
+import {
+    IAdvancedSearchStyleProps,
+    IAdvancedSearchStyles
+} from './AdvancedSearch.types';
+
+export const classPrefix = 'cb-advancedsearch';
+const classNames = {
+    content: `${classPrefix}-content`,
+    header: `${classPrefix}-header`,
+    headerText: `${classPrefix}-headerText`,
+    queryContainer: `${classPrefix}-queryContainer`,
+    resultsContainer: `${classPrefix}-resultsContainer`
+};
+
+export const getStyles = (
+    _props: IAdvancedSearchStyleProps
+): IAdvancedSearchStyles => {
+    return {
+        content: [classNames.content],
+        header: [
+            classNames.header,
+            {
+                display: 'flex',
+                // TODO: Remove this margin when modal contents are done
+                marginBottom: 150
+            }
+        ],
+        headerText: [
+            classNames.headerText,
+            {
+                margin: 0
+            }
+        ],
+        queryContainer: [
```

Comment:
I would put the styles for the list and such in it's own file since it's kind of it's own thing. Doensn't have to be for this PR though.